src/js/src/components/LocaleChanger.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="localeChanger">
+    <div class="localeChangerContainer">
+      <a href="javascript:void(0);" @click.prevent="toggleLang">
+        {{ $t('solrWayback:toggle_language') }}: <span v-html="lang"></span>
+      </a>
```

Comment:
Nice with a toggler! 

Consider turning this into a 'select' so it supports multiple languages. 

If the conversion to select is not doable right now then consider using '#' in the href to avoid triggering Content Security Policies violation warnings ( https://developer.mozilla.org/en/docs/Security/CSP ) on CSP-enabled HTTPS sites. You already have the 'prevent default' attribute set.